src/NuGet.Config

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
`git diff` on the commandline is helpful for this:

![image](https://cloud.githubusercontent.com/assets/1376924/8566821/1b0db398-2561-11e5-9b0b-fa3dce6e13c8.png)


Looks like you changed the BOM.